builds/misc/images-release.yaml

```diff
@@ -0,0 +1,602 @@
+                    {
+                        "parameterName": "PageHash",
+                        "parameterValue": "/NPH"
+                    },
+                    {
+                        "parameterName": "TimeStamp",
+                        "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+                    }
+                    ],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                },
+                {
+                    "keyCode": "CP-230012",
+                    "operationSetCode": "SigntoolVerify",
+                    "parameters": [
+                    {
+                        "parameterName": "VerifyAll",
+                        "parameterValue": "/all"
+                    }
+                    ],
+                    "toolName": "sign",
+                    "toolVersion": "1.0"
+                }
+              ]
+          signConfigType: inlineSignParams
+      - task: PowerShell@2
+        displayName: "Build Image - Edge Agent - amd64"
+        inputs: 
+          arguments: "-Name 'azureiotedge-agent' -Project 'Microsoft.Azure.Devices.Edge.Agent.Service' -Version '$(version)' -Registry '$(registry.address)' -Namespace '$(namespace)' -Push"
+          filePath: ./scripts/windows/build/Publish-DockerImage.ps1
+          targetType: filePath
+      - task: PowerShell@2
+        displayName: "Build Image - Edge Hub - amd64"
+        inputs: 
+          arguments: "-Name 'azureiotedge-hub' -Project 'Microsoft.Azure.Devices.Edge.Hub.Service' -Version '$(version)' -Registry '$(registry.address)' -Namespace '$(namespace)' -Push"
+          filePath: ./scripts/windows/build/Publish-DockerImage.ps1
+          targetType: filePath
+      - task: PowerShell@2
+        displayName: "Build Image - Temp Sensor - amd64"
+        inputs: 
+          arguments: "-Name 'azureiotedge-simulated-temperature-sensor' -Project 'SimulatedTemperatureSensor' -Version '$(version)' -Registry '$(registry.address)' -Namespace '$(namespace)' -Push"
+          filePath: ./scripts/windows/build/Publish-DockerImage.ps1
+          targetType: filePath
+      - task: PowerShell@2
+        displayName: "Build iotedge-diagnostics"
+        inputs:
+          filePath: ./edgelet/build/windows/build-diagnostics.ps1
+          targetType: filePath
+      - task: PowerShell@2
+        displayName: "Build Image - azureiotedge-diagnostics - amd64"
+        inputs: 
+          arguments: "-Name 'azureiotedge-diagnostics' -Project 'azureiotedge-diagnostics' -Version '$(version)' -Registry '$(registry.address)' -Namespace '$(namespace)' -Push"
+          filePath: ./scripts/windows/build/Publish-DockerImage.ps1
+          targetType: filePath
+      - task: PublishBuildArtifacts@1
+        displayName: "Publish Artifacts to VSTS"
+        inputs: 
+          ArtifactName: publish-win
+          PathtoPublish: $(Build.BinariesDirectory)\publish
```

Comment:
Why only do this for windows?

Reply:
Is there any reason not to do this for Linux? Does it just publish to the UI on VSTS? If so, it seems fine to either do it on Linux too or not do it for either, since we smoke test by pulling from EdgeRelease anyway

Reply:
Ok I published for both

Reply:
I searched PublishBuildArtifacts@1 task but it is only used in Windows job, but not in Linux job, are you going to add it?

Reply:
Thanks pushed to wrong branch. Added.